worker/transcribee_worker/identify_speakers.py

```diff
@@ -12,10 +11,8 @@
 from .config import settings
 
 
-async def identify_speakers(
-    audio: npt.NDArray, doc: Document, progress_callback: Callable[[str, float], None]
-):
-    def work(queue):
+async def identify_speakers(audio: npt.NDArray, doc: Document, progress_callback):
```

Comment:
Can I have typings for the callback please? Ideally one that even contains the allowed kwargs?